packages/connect-ui/src/views/IntegrationsList.tsx

```diff
@@ -51,20 +53,28 @@ const Integrations: React.FC = () => {
 };
 
 const Integration: React.FC<{ integration: ApiPublicIntegration }> = ({ integration }) => {
+    const store = useGlobal();
+    const navigate = useNavigate();
+
     const [loading, setLoading] = useState(false);
+    const [error, setError] = useState<string | null>(null);
 
     async function triggerAuth() {
         setLoading(true);
+
         let provider: GetPublicProvider['Success'] | undefined;
         try {
             provider = await getProvider({ provider: integration.provider });
         } catch (err) {
-            console.log(err); // TODO: handle this
+            console.log(err);
```

Comment:
is this still needed?

Reply:
Since we don't show actual error, I'm usually fine to log errors in the console so we can open the console in prod. wdyt?